tests/hw-accel/amdgpu/internal/pods/pods.go

```diff
@@ -143,3 +143,35 @@ func WaitUntilNoMorePodsInNamespaceByNameWithTimeout(ctx context.Context, apiCli
 		}
 	}
 }
+
+// GetPodsFromNamespaceByPrefixWithTimeout - Gets all pods in a namespace whose names start with a specified prefix.
+func GetPodsFromNamespaceByPrefixWithTimeout(
```

Comment:
I would suggest creating a `get` package and rename this to `PodsFromNamespaceByPrefixWithTimeout`

This way it should be called like `get.PodsFromNamespaceByPrefixWithTimeout`

The example from KMM is here: https://github.com/rh-ecosystem-edge/eco-gotests/tree/main/tests/hw-accel/kmm/internal